modules/lazyload/module.php

```diff
@@ -131,9 +132,17 @@ public function __construct() {
 						}
 					});
 					}, { rootMargin: '100px 0px 100px 0px' } );
-					lazyloadBackgrounds.forEach( ( lazyloadBackground ) => {
-						lazyloadBackgroundObserver.observe( lazyloadBackground );
+						lazyloadBackgrounds.forEach( ( lazyloadBackground ) => {
+							lazyloadBackgroundObserver.observe( lazyloadBackground );
 					} );
+
+				};
+				const events = [
+					'DOMContentLoaded',
+					'elementor/widgets/posts/load-more',
```

Comment:
Are you sure that this event it running? (it doesn't for me).

Reply:
Yes, just load a page with lazy load background, then remove the `lazyloaded` class.
Now, you can run to console the event :

`document.dispatchEvent( new CustomEvent( 'elementor/widgets/posts/load-more' ) );`

and you will see the background image appear